src/databricks/labs/ucx/source_code/queries.py

```diff
@@ -16,7 +16,7 @@ class FromTable(Linter, Fixer):
     SQL queries.
     """
 
-    def __init__(self, index: MigrationIndex, session_state: CurrentSessionState):
+    def __init__(self, index: MigrationIndex, session_state: CurrentSessionState | None = None):
```

Comment:
it must not be None, because for redash it's possible to set a current schema name in the query:
<img width="397" alt="image" src="https://github.com/databrickslabs/ucx/assets/259697/a3310680-9b3c-4005-8533-cea24a9358a0">

these can be retrieved it from `query` object `options` -> `catalog` and `schema` fields